README.md

```diff
@@ -1,2 +1,72 @@
-# infrastructure
-Infrastructure as a Code for Open Power HostOS 
+# OpenPOWER Host OS infrastructure as a code
+
+This repository keeps automated scripts to set up an infrastructure capable of
+building OpenPOWER Host OS.
+
+## Prepare CentOS system
+
+First, you'll need to prepare at least one CentOS system. This could be
+bare-metal, virtual machines or containers.
+
+## Execute Ansible playbooks
+
+There are Ansible playbooks that set up the Jenkins master node and two Jenkins
+slave nodes, one for Host OS builds and one for validating the code from GitHub
+pull requests. There could be multiple slave nodes set up for parallel builds.
+The different roles (master, builds slave and validation slave) could even be
+set up in the same system, as desired.
+
+Read the instructions at ansible/README for details on how to execute the
+Ansible playbooks.
+
+Note: The "master" playbook may fail due to Jenkins temporary unresponsiveness.
+If you see HTTP request errors, try running it again.
+
+## Manual Jenkins steps
+
+### Seed job
+
+Open the server's webpage in your browser. It should show the Jenkins webpage,
+with a single job: "seed_job". Execute it, making sure the parameters are
+adequate for you infrastructure. With the default parameters, this will create
+all the other jobs configured to check out from the GitHub open-power-host-os
```

Comment:
Can we add a link to https://github.com/open-power-host-os/ where it says open-power-host-os?  Something like `[open-power-host-os](https://github.com/open-power-host-os/)`.